packages/block-library/src/details-content/edit.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * WordPress dependencies
+ */
+import { useBlockProps, useInnerBlocksProps } from '@wordpress/block-editor';
+import { __ } from '@wordpress/i18n';
+
+const TEMPLATE = [
+	[
+		'core/paragraph',
+		{
+			placeholder: __(
+				'Add text or blocks that will display when the details block is opened.'
+			),
+		},
+	],
+];
+
+function DetailsContentBlock() {
```

Comment:
```suggestion
function DetailsContentEdit() {
```

Nit: The naming convention for core block edit components appears to be `<BlockName>Edit`. At least the first dozen or so blocks I checked followed this. Most likely as it more accurately represents the component in this `edit.js` file.